rootfs/etc/cont-init.d/udev.sh

```diff
@@ -2,7 +2,7 @@
 # ==============================================================================
 # Start udev service if env USING_UDEV is true
 # ==============================================================================
-if ! bashio::var.true "${USING_UDEV}"; then
+if ! bashio::var.true "${USING_UDEV:-}"; then
```

Comment:
```suggestion
if ! bashio::var.true "${USING_UDEV+x}"; then
```

Reference:
- https://github.com/home-assistant/core/pull/36274
- https://stackoverflow.com/questions/48926946/what-does-variablex-mean-in-bash